backend/entities/LeasePayment.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations.Schema;
+namespace Pims.Dal.Entities
+{
+    /// <summary>
+    /// PimsLeasePayment class, provides an entity for the datamodel to manage lease payments.
+    /// </summary>
+    public partial class PimsLeasePayment : IdentityBaseAppEntity<long>, IBaseAppEntity
+    {
+        #region Properties
+        [NotMapped]
+        public override long Id { get => this.LeasePaymentId; set => this.LeasePaymentId = value; }
```

Comment:
this looks a tad hackish

Reply:
alternatives?

Reply:
Maybe I am missing something, is this only used by the UpdateChild method?